Jenkinsfile

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'service-manual-publisher'
+
+// @todo Replace with govuk.setupDb when on Rails 5.
+def setupDb() {
```

Comment:
On the off-chance that we remain on Rails 4.x for longer than anticipated, can we add a comment to explain _why_ we need this because we're not on Rails 5?